tests/test_views.py

```diff
@@ -2848,25 +2848,6 @@ def _make_pointer_only_user_can_see(self, user, project, save=False):
         node = ProjectFactory(creator=user)
         project.add_pointer(node, auth=Auth(user=user), save=save)
 
-    def test_pointer_list_write_contributor_can_remove_private_component_entry(self):
```

Comment:
Would https://github.com/CenterForOpenScience/osf.io/pull/5860/commits/c3b4a96d779e4f61ab0a5037c08bf87ec6c43794 make this pass again?